README.md

```diff
@@ -0,0 +1,83 @@
+# STOCKIST WEB APP - REACT CAPSTONE PROJECT
+
+The project, a frontend web application, catalogue of financial values using [Stock API](https://financialmodelingprep.com/developer/docs/)  and collect the data from my React Application.
+
+The application features includes:
+- Listing of Most Active Stocks
+- Listing of Most Gainer's Stocks
+- Listing of Most Loser's Stocks
+- Filter between those 3, most active, gainers and losers
+- Open individual stock item
+- Search for a particular stock item, example 'AAPL'
+- Listing of the lastest stock news
+- Dynamic routes for individual stock item, example `/stocks/AAPL`
+
+![screenshot](./stockist.gif)
+
+## Live Demo
+[Live Demo](https://634140379320270009398402--tranquil-gumdrop-ab7d6d.netlify.app)
+
+## Video 
+[Video](https://drive.google.com/file/d/1bhOM4IuZjATnMbhVdWN6Egx35ny3SPa1/view?usp=sharing)
```

Comment:
- [ ] It's recommended that students use [Loom](https://www.loom.com/) to record the video. Since you have decided to use other tools, please make sure that we have full access to the video. You can also upload it to Youtube. Below is the screenshot of the error I got when I click on this link

![Screenshot from 2022-10-08 11-56-04](https://user-images.githubusercontent.com/63087491/194704139-55a54d3e-2362-4ff4-a5ef-805a468a38bc.png)

Reply:
@FrederickMih The video presentation is working it works on my side